spec/parliament/utils_spec.rb

```diff
@@ -22,23 +22,23 @@
 
```

Comment:
Should these utils be part of the Ntriple gem? It's directly related to that data and output?